src/nvim/tui/tui.c

```diff
@@ -1190,7 +1199,8 @@ static void unibi_goto(UI *ui, int row, int col)
       str = fn(data->ut, (unsigned)unibi_index); \
     } \
     if (str) { \
-      unibi_var_t vars[26 + 26] = { { 0 } }; \
+      unibi_var_t vars[26 + 26]; \
+      memset(&vars, 0, sizeof(vars)); \
```

Comment:
how's this different/better than `= { 0 }` ?

Reply:
It doesn't trigger the deprecation warning.